### PR TITLE
feat/ COR-1468 archived page and archived metrics

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard27**
## Summary

* Moved 'kwetsbare-groepen-70-plussers' page in the archived pages section.
* Changed the section sub-title.
* Added warning tile.
* Archived page metrics.
* Removed page information block and divider. 

#### Before
<details>
<summary>Toggle before screenshots</summary>

Screenshot 2023-07-10 at 17 05 25

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

Screenshot 2023-07-10 at 17 04 30

</details>

### CODE REVIEW

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Looks awesome!! Great job!

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Nice! Good catch on deleting the Information Block!

File: packages/app/schema/archived_nl/nursing_home.json
Comment (VWSCoronaDashboard28) [inline packages/app/schema/archived_nl/nursing_home.json:1]: In this file, the `title` field contains the `archived_[DATE]` suffix. If I'm not wrong, I don't think we usually add this suffix there so I would suggest to remove it here, and the generate the data types again.

File: packages/common/src/types/data.ts [hunk 1]
```diff
@@ -54,11 +54,27 @@ export interface ArchivedNl {
   proto_name: ArchivedNlId;
   name: ArchivedNlId;
   code: ArchivedNlId;
+  difference: ArchivedNlDifference;
   behavior_archived_20230411: NlBehavior;
   behavior_annotations_archived_20230412: NlBehaviorAnnotations;
   behavior_per_age_group_archived_20230411: NlBehaviorPerAgeGroup;
   doctor_archived_20210903: NlDoctor;
+  nursing_home_archived_20230126: NlNursingHomeArchived_20230126;
```
Comment (VWSCoronaDashboard28) [inline packages/common/src/types/data.ts:62]: When you change the `title` in `nursing_home.json` as mentioned in my other comment, then `NlNursingHomeArchived_20230126` should change as well.

**Review by VWSCoronaDashboard28 — APPROVED** [review]
Looks good, just a couple of small comments.

File: packages/app/schema/archived_nl/nursing_home.json
Comment (VWSCoronaDashboard28) [reply packages/app/schema/archived_nl/nursing_home.json:1]: It says this is outdated but it is still relevant

File: packages/common/src/types/data.ts (hunk 1 above)
Comment (VWSCoronaDashboard27) [reply packages/common/src/types/data.ts:62]: @VWSCoronaDashboard28  Isn't it only value and last value supposed to change after removing suffix from nursing home.json?

Comment (VWSCoronaDashboard28) [reply packages/common/src/types/data.ts:62]: Please see my other previous comment. We can have a call if its not clear.

File: packages/app/schema/archived_nl/nursing_home.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
   "additionalProperties": false,
   "definitions": {
     "value": {
-      "title": "nl_nursing_home_archived_20230126_value",
+      "title": "nl_nursing_home_value",
```
Comment (VWSCoronaDashboard28) [inline packages/app/schema/archived_nl/nursing_home.json:20]: I think you may have misunderstood my comment. I meant the title of the Schema should change. So the second line in this file.

Comment (VWSCoronaDashboard28) [reply packages/app/schema/archived_nl/nursing_home.json:20]: This line should be reverted.

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Thumbs up!

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Review by VWSCoronaDashboard21 — APPROVED** [review]

**Pull request merged**